doc/user/builders-writing.xml

```diff
@@ -43,78 +43,6 @@
   OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
   WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
--->
```

Comment:
I think worth leaving this in commented out for now.
It's been copied from cons and may have value.. if we get a chance to implement?

Reply:
> I think worth leaving this in commented out for now.
> It's been copied from cons and may have value.. if we get a chance to implement?

I'll of course put it back if you want, it's your call, and costs basically nothing to do.  But I think this kind of wishlist-item works better stored somewhere else. It's been 20 years, and at current contribution levels implementation seems... unlikely. I'm hereby expressing my preference for the docs to be cleaner, not cluttered with "what if someday?". Wiki perhaps?

Reply:
Maybe move to github issue as enhancement?
That way it's not lost, but doesn't clutter up the actual docs.

Reply:
I'm fine with that, as long as you don't worry about issues being cluttered with inactive stuff (have noticed this only seems to both the maintainer with PRs, not issues :) )

Reply:
I'm fine with having many enhancements filed, we can add to projects like "wishlist 1".. or such.
It's really the # of non-enhancement issues (aka bugs) IMHO which are more problematic.

Reply:
Parked in #3887.